scripts/vitestSetup.ts

```diff
@@ -24,48 +24,61 @@ export function slash(p: string): string {
   return p.replace(/\\/g, '/')
 }
 
-// injected by the test env
-declare global {
-  const page: Page | undefined
-
-  const browserLogs: string[]
-  const browserErrors: Error[]
-  const serverLogs: string[]
-  let viteTestUrl: string | undefined
-  const watcher: RollupWatcher | undefined
-  let beforeAllError: Error | null // error caught in beforeAll, useful if you want to test error scenarios on build
-}
-
-declare const global: {
-  page?: Page
-
-  browserLogs: string[]
-  browserErrors: Error[]
-  serverLogs: string[]
-  viteTestUrl?: string
-  watcher?: RollupWatcher
-  beforeAllError: Error | null
-}
+// // injected by the test env
+// declare global {
+//   const page: Page | undefined
+
+//   const browserLogs: string[]
+//   const browserErrors: Error[]
+//   const serverLogs: string[]
+//   let viteTestUrl: string | undefined
+//   const watcher: RollupWatcher | undefined
+//   let beforeAllError: Error | null // error caught in beforeAll, useful if you want to test error scenarios on build
+// }
+
+// declare const global: {
+//   page?: Page
+
+//   browserLogs: string[]
+//   browserErrors: Error[]
+//   serverLogs: string[]
+//   viteTestUrl?: string
+//   watcher?: RollupWatcher
+//   beforeAllError: Error | null
+// }
```

Comment:
Do we need to keep these commented out code?

Reply:
Oh I forget to remove them